scripts/Patches.py

```diff
@@ -427,4 +427,10 @@ def patch_rom(rom:Rom, options):
     save_context.write_save_table(rom)
 
     rom.write_int32(rom.sym('CFG_DUNGEON_INFO_ENABLE'), 1)
+
+    if options['spoiler']:
```

Comment:
I'm okay with the spoiler being in the base rom now that we've established the "SL" convention.